competition/evaluation/evaluate.py

```diff
@@ -15,6 +15,7 @@
     "eval_episodes",
     "seed",
     "scenarios",
+    "bubble_env_evaluation_seeds",
```

Comment:
Consider using short variable names. Here, we could use `bubble_env_seeds` instead of `bubble_env_evaluation_seeds`.